stories/core/Buttons/ButtonGroup.stories.tsx

```diff
@@ -37,3 +54,61 @@ export const WithIcons: Story<ButtonGroupProps> = () => {
     </ButtonGroup>
   );
 };
+
+export const Overflow: Story<ButtonGroupProps> = (args) => {
+  const buttons = Array(10)
+    .fill(null)
+    .map((_, index) => (
+      <IconButton
+        key={index}
+        onClick={() => action(`Clicked on button ${index + 1}`)()}
+      >
+        <SvgPlaceholder />
+      </IconButton>
+    ));
+
+  return (
+    <div
+      style={{
+        maxWidth: 'clamp(300px, 50%, 100%)',
+        border: '1px solid hotpink',
+        padding: 8,
+      }}
+    >
+      <ButtonGroup
+        style={{ display: 'flex' }}
+        overflowButton={(overflowStart) => (
+          <DropdownMenu
+            menuItems={(close) =>
+              Array(buttons.length - overflowStart + 1)
+                .fill(null)
+                .map((_, _index) => {
+                  const index = overflowStart + _index;
+                  const onClick = (close: () => void) => {
```

Comment:
You don't really need to have `close` as a param.

Reply:
?